master/usage/ipv6.md

```diff
@@ -1,20 +1,155 @@
 ---
-title: IPv6 Support
+title: Enabling IPv6 Support
 canonical_url: 'https://docs.projectcalico.org/v3.0/usage/ipv6'
 ---
 
-{{site.prodname}} supports connectivity over IPv6, between compute hosts, and
-between compute hosts and their VMs. This means that, subject to
-security configuration, a VM can initiate an IPv6 connection to another
-VM, or to an IPv6 destination outside the data center; and that a VM can
-terminate an IPv6 connection from outside.
+{{site.prodname}} supports connectivity over IPv6, for hosts and
+their workloads. This means that, subject to security configuration,
+a workload can initiate an IPv6 connection to another
+workload, or to an IPv6 destination outside the data center; and that a
+workload can terminate an IPv6 connection from outside.
+
+
+## Enabling IPv6 with Kubernetes
+
+Currently Kubernetes supports only one IP stack version at a time. This means
```

Comment:
I think we should try to organize this sort of information into either a "Prerequisites" or "Limitations" section towards the top of the page, for reasons similar to what I mention in my comment above.

In doing so, it would be great if we could break it down from large paragraphs of text into shorter bullet points - that makes it a lot easier for people to parse.